include/cinder/Log.h

```diff
@@ -135,18 +140,46 @@ class LoggerBreakpoint : public Logger {
 	Level	mTriggerLevel;
 };
 
-#if defined( CINDER_COCOA )
-
-class LoggerSysLog : public Logger {
+class LoggerSystem : public Logger {
 public:
-	LoggerSysLog();
-	virtual ~LoggerSysLog();
-
-	void write( const Metadata &meta, const std::string &text ) override;
-};
-
+	LoggerSystem();
+	virtual ~LoggerSystem();
+	
+	void write( const Metadata &meta, const std::string &text ) override { mImpl->write( meta, text ); }
+	// SysLog and EventLog logging levels inherently work differently (see specific comments).  Currently
+	// this doesn't matter since logging implimentation only allows once instance of a logger type.
```

Comment:
How do they work differently? Not sure what the user should take away from this message, or if there are functionality differences in practice.